content/api/commands/invoke.md

```diff
@@ -26,15 +35,19 @@ subject, use [`.its()`](/api/commands/its).
 **<Icon name="check-circle" color="green"></Icon> Correct Usage**
 
 ```javascript
-cy.wrap({ animate: fn }).invoke('animate') // Invoke the 'animate' function
+cy.get('.input').invoke('val').should('eq', 'foo') // Invoke the 'val' function
```

Comment:
given the alert above, wouldn't `invoke('val)` run 1-many times until should is satisfied or it times out? Is this the expected usage of invoke?

Reply:
Yes, this is the expected usage of invoke, and one of the most common ways people use it (as measured via querying metabase test bodies).

"Get an input, invoke the `val` method on the jquery object, assert that the value matches a string" is :+1: